src/App.tsx

```diff
@@ -8,7 +8,56 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { Todo } from './types/Todo';
+
+import { getTodos } from './api';
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [userId, setUserId] = useState<number>(0);
+  const [query, setQuery] = useState<string>('');
```

Comment:
don't need the generic here, actually
it is automatically calculated based on the default value
```suggestion
  const [query, setQuery] = useState('');
```